src/widgets/helper/ChannelView.hpp

```diff
@@ -192,6 +209,12 @@ class ChannelView final : public BaseWidget
                          QPoint &relativePos, int &index);
 
 private:
+    struct InternalCtor {
+    };
+
+    ChannelView(InternalCtor tag, QWidget *parent, Split *split,
+                Context context = Context::None, size_t messagesLimit = 1000);
```

Comment:
We can remove all the default values here
```suggestion
    ChannelView(InternalCtor tag, QWidget *parent, Split *split,
                Context context, size_t messagesLimit);
```